<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharGen2</class>
 <widget class="QWidget" name="CharGen2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>460</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="standardMode">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>275</y>
     <width>175</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Schoolbook L</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(136, 138, 133);</string>
   </property>
   <property name="text">
    <string>Стандартный режим</string>
   </property>
  </widget>
  <widget class="QPushButton" name="randomMode">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>275</y>
     <width>175</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Schoolbook L</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(136, 138, 133);</string>
   </property>
   <property name="text">
    <string>Случайный режим</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="nextStep">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>385</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Schoolbook L</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(136, 138, 133);
color: white;</string>
   </property>
   <property name="text">
    <string>Далее</string>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="chpic">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>35</y>
     <width>160</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
